devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -300,6 +303,22 @@ def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulat
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	def checkVal
+	// the last message sent by configure is a configuration get, so if we get a report, we succeeded in transmission
+	// and if the parameter 1 values match what we expect, then the configuration probably succeeded
+	if (isZipato()) {
+		checkVal = 3
```

Comment:
3?

Reply:
I guess that is what we are setting this to in configuration.

Reply:
I feel like maybe we should define these somewhere so that they aren't magic numbers. I can see someone changing the value in one place but not elsewhere. Maybe as `def getZipatoVolume() { 3 }`, `def getZipatoDuration() { 2 }`, `def getZipatoSound() { 10 }`?

Reply:
magic numbers removed